bower.json

```diff
@@ -2,7 +2,7 @@
     "name": "adapt-hotspot-menu",
     "repository": "git://github.com/cgkineo/adapt-hotspot-menu",
     "framework": "^2.0.0",
-    "version": "2.0.0",
+    "version": "2.0.13",
```

Comment:
why the massive jump in version number?

Reply:
Probably because as it was developed locally and in the contentteam adapt instance. I realise that I could have flip-flopped between 2.0.0 and 2.0.1! Can reduce to 2.0.1, though given @tomgreenfield 's comments below I'm not so sure now.

Reply:
TBH the amount of changes in this PR make me think it needs bumping to 2.1.0 - @tomgreenfield what do you think?